documentation/user-interface/explorer-panel.md

```diff
@@ -67,9 +61,9 @@ To run several tests from different fixtures and directories, select the tests u
 
 You can also use the context menu's ![Run test icon](../../images/user-interface/context-menu/run-tests.png) **Run Test** command to run tests.
 
-## Modify a Fixture
+## Modify a Fixture's Properties
```

Comment:
Extra article

Reply:
no, this article is required:
- whose properties?  
- one fixture.